pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -120,8 +120,8 @@ let
 
   useGccForLibs = useCcForLibs
     && libcxx == null
+    && targetPlatform.rtlib == "libgcc"
     && !targetPlatform.isDarwin
```

Comment:
Can the Darwin check be dropped since `targetPlatform.rtlib` will be `compiler-rt` on Darwin.

Reply:
If someone changes it to `libgcc` on Darwin?